cli/ops/plugins.rs

```diff
@@ -77,27 +55,13 @@ pub fn op_open_plugin(
     .get_mut::<PluginResource>(rid)
     .unwrap();
 
-  let init_fn = *unsafe {
+  let deno_plugin_init = *unsafe {
     plugin_resource
       .lib
       .symbol::<PluginInitFn>("deno_plugin_init")
-  }?;
-  let mut init_context = InitContext {
-    ops: HashMap::new(),
-  };
-  init_fn(&mut init_context);
-  for op in init_context.ops {
-    // Register each plugin op in the `OpRegistry` with the name
-    // formated like this `plugin_{plugin_rid}_{name}`.
-    // The inclusion of prefix and rid is designed to avoid any
-    // op name collision beyond the bound of a single loaded
-    // plugin instance.
-    let op_id = isolate
-      .register_op(&format!("plugin_{}_{}", rid, op.0), state.core_op(op.1));
-    plugin_resource.ops.insert(op.0, op_id);
```

Comment:
This bit was used to make sure there are no ops with duplicate name - right now program will panic if you try to open a plugin that registers op with already existing name

Reply:
That's right